client/src/components/charts/MapVisualisation.jsx

```diff
@@ -25,8 +26,11 @@ export default function MapVisualisation({ visualisation, datasets, width, heigh
           attribution="&copy; <a href='http://osm.org/copyright'>OpenStreetMap</a> contributors"
         />
         {
-          chartData[0].values.map((entry, index) =>
-            <Marker position={entry.position} key={index}>
+          chartData.values.map((entry, index) =>
+            <Marker
+              position={[parseFloat(entry.x), parseFloat(entry.y)]}
```

Comment:
At some point we should consider not parsing when rendering charts, as it's not really necessary if we only show numeric columns for properties that needs to be numeric.